db/re/mob_db.yml

```
@@ -78190,6 +78190,9 @@ Body:
     DamageMotion: 768
     Ai: 10
     Class: Boss
+    Modes:
+      CanMove: true
+      Detector: true
```

Class 'boss' already include detector. I guess the 'Ai 'should be 21 lika all other bosses? (same for other corrections)
```suggestion
    Ai: 21
    Class: Boss
```